src/aiida_quantumespresso/workflows/functions/get_xspectra_structures.py

```diff
@@ -15,6 +15,160 @@
 from aiida_quantumespresso.utils.hubbard import HubbardStructureData, HubbardUtils
 
 
+def process_molecule_input(structure, **kwargs):  # pylint: disable=too-many-statements
+    """Generate symmetry data and supercell suitable for molecular systems using Pymatgen."""
+    from pymatgen.symmetry.analyzer import PointGroupAnalyzer
+
+    result = {'output_params': {}}
+    # If we are working with a molecule, check for pymatgen_settings
```

Comment:
```suggestion

```

Reply:
Could you also fix this? The structure is a `molecule` inside this function.

Reply:
Sorry about that. I thought I'd removed the comment in the first set of changes that I committed. It's been removed in the [latest commit](https://github.com/aiidateam/aiida-quantumespresso/pull/1026/commits/fbba65160a15df4399c1d74e025eb0a046201c6c)